test_data/makedata_suites/403_utf8-name-views.js

```diff
@@ -0,0 +1,67 @@
+/* global print, progress, createCollectionSafe, db, createSafe, semver  */
+
+(function () {
+  let extendedNames = ["ᇤ፼ᢟ⚥㑸ন", "に楽しい新習慣", "うっとりとろける", "זַרקוֹר", "ስፖትላይት", "بقعة ضوء", "ուշադրության կենտրոնում", "🌸🌲🌵 🍃💔"];
+  return {
+    isSupported: function (currentVersion, oldVersion, options, enterprise, cluster) {
+      let currentVersionSemver = semver.parse(semver.coerce(currentVersion));
+      let oldVersionSemver = semver.parse(semver.coerce(oldVersion));
+      return semver.gte(currentVersionSemver, "3.11.0") && semver.gte(oldVersionSemver, "3.11.0");
+    },
+    makeData: function (options, isCluster, isEnterprise, dbCount, loopCount) {
+      // All items created must contain dbCount and loopCount
+      print(`403: making data ${dbCount} ${loopCount}`);
+      let viewCollectionName = `old_cview1_${loopCount}${extendedNames[3]}`;
+      let cview1 = createCollectionSafe(viewCollectionName, 3, 1);
+      progress('403: createView1');
+      let viewName1 = `old_view1_403_${loopCount}${extendedNames[6]}`;
+      let view1 = createSafe(viewName1,
```

Comment:
Along with `arangosearch` view, we should test `search-alias` as well

Reply:
QA-457 added